doc/nrf/gs_modifying.rst

```diff
@@ -52,8 +75,66 @@ The following CMake commands can be managed by SES, if they target the ``app`` l
     - ``target_compile_options``
 
 The :file:`CMakeLists.txt` files for the sample applications in the |NCS| are tagged as required.
-Typically, they include at least the :file:`main.c` file as source::
+Therefore, if you always use SES to maintain them, you do not need to worry about tagging.
+Typically, the :file:`CMakeLists.txt` files include at least the :file:`main.c` file as source::
 
    # NORDIC SDK APP START
    target_sources(app PRIVATE src/main.c)
    # NORDIC SDK APP END
+
+
+Configuring your application
+****************************
+
+If your application uses a provided library or targets a specific board, you might want to change the default configuration of the library or board.
+There are different ways of doing this, but not all will store your configuration permanently.
+
+The default configuration for a library is stored in its :file:`Kconfig` file.
```

Comment:
I wouldn't use the word 'stored', as it implies there is some state there, perhaps specified, declared, or described.